syntaxes/graphql.js.json

```
@@ -14,7 +14,7 @@
           "name": "punctuation.definition.string.template.begin.js"
         }
       },
-      "end": "`",
+      "end": "`\\(?",
```

Did you mean a closing parenthesis here?

```suggestion
      "end": "`\\)?",
```